WikiWatershed/model-my-watershed
# Update PA DEP Excel Spreadsheet to latest

## Overview

Previous version had some default values in other places.
This version is clean. Corresponds to https://github.com/WikiWatershed/MMW-BMP-spreadsheet-tool/blob/v1.0.0/MMW_BMP_Spreadsheet_Tool.xlsx

Connects #3235

### Demo

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/1430060/72938405-ece33e80-3d38-11ea-80d4-02781dcd4d55.png">

###

This new version has been deployed "manually" to production by copying the template to all the app servers and running

```console
$ sudo service mmw-app stop
$ sudo service mmw-app start
```

on them, and that has been verified by BME.

## Testing Instructions

* Check out this branch
* Go to [:8000/](http://localhost:8000/) and draw a shape
* Generate a PA DEP worksheet
* In the downloaded Excel file, go to the "Urban BMPs" tab
  - [ ] Ensure there are no pre-filled values there
